### ISSUE TITLE
Autor field must be optional

### ISSUE THREAD

**Issue by GrakovNe** [GrakovNe/lissen-android#32]
Error from Acra:

```
Stacktrace (No mapping found)
java.lang.NullPointerException: Parameter specified as non-null is null: method org.grakovne.lissen.domain.Book.<init>, parameter author
	at org.grakovne.lissen.domain.Book.<init>(Unknown Source:12)
	at org.grakovne.lissen.channel.audiobookshelf.converter.LibraryItemResponseConverter.apply(LibraryItemResponseConverter.kt:16)
	at org.grakovne.lissen.channel.audiobookshelf.AudiobookshelfChannel$fetchBooks$2.invokeSuspend(AudiobookshelfChannel.kt:90)
	at org.grakovne.lissen.channel.audiobookshelf.AudiobookshelfChannel$fetchBooks$2.invoke(Unknown Source:8)
	at org.grakovne.lissen.channel.audiobookshelf.AudiobookshelfChannel$fetchBooks$2.invoke(Unknown Source:4)
	at org.grakovne.lissen.channel.common.ApiResult.map(ApiResult.kt:29)
	at org.grakovne.lissen.channel.audiobookshelf.AudiobookshelfChannel.fetchBooks(AudiobookshelfChannel.kt:90)
	at org.grakovne.lissen.channel.audiobookshelf.AudiobookshelfChannel$fetchBooks$1.invokeSuspend(Unknown Source:16)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8592)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
```

**Comment by GrakovNe** [comment]
https://github.com/GrakovNe/lissen-android/pull/33

**Comment by GrakovNe** [comment]
Fixed & tested in 1.0.21
